examples/weather/unified_recipe/seq_zarr_datapipe.py

```diff
@@ -73,23 +73,35 @@ class SeqZarrDatapipe(Datapipe):
 
     def __init__(
         self,
-        zarr_dataset: zarr.hierarchy.Group,
+        file_mapping: str,
```

Comment:
still typed as `str`?

Reply:
Woops, fixed it. There was another mistake in the external source too.